package-lock.json

```diff
@@ -34,7 +34,7 @@
       }
     },
     "extension": {
-      "name": "designer-11",
+      "name": "designer-13",
```

Comment:
why version needed?

Reply:
in order to make this thing unique

Reply:
see id:
![image](https://github.com/user-attachments/assets/48429f35-2e78-48c3-9fbe-bf9642a9678e)